### PR TITLE
ci(pre-commit): clear the exclude option

### PR DESCRIPTION
**Pull request by kenji-miyake**
I think the packages that were added to `exclude` before should be removed since we've finished porting them and there is no more sync with Autoware.Auto.

Related: https://github.com/autowarefoundation/autoware.universe/issues/248

### CODE REVIEW

**Comment by kenji-miyake** [comment]
@TakaHoribe @maxime-clem Do you have any concerns about this?

**Comment by maxime-clem** [comment]
> @TakaHoribe @maxime-clem Do you have any concerns about this?

No concern from me but the packages that were ignored now need to be formatted properly.
Let me know if I can help you with that.

**Review by TakaHoribe — DISMISSED** [review]
LGTM. I don't have any concerns.

**Comment by kenji-miyake** [comment]
@xmfcx @mitsudome-r @angry-crab Please let me know if there are any concerns. If not, I'll fix CI errors and merge this. :bow:

**Comment by xmfcx** [comment]
I am also ok with these changes.

File: common/vehicle_constants_manager/package.xml [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
+  <depend>autoware_auto_common</depend>
   <depend>rclcpp</depend>
   <depend>rclcpp_components</depend>
-  <depend>autoware_auto_common</depend>
 
   <test_depend>ament_cmake_gtest</test_depend>
-  <!-- <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend> -->
   <test_depend>autoware_auto_cmake</test_depend>
+  <!-- <test_depend>ament_lint_auto</test_depend>
```
Comment (kenji-miyake) [inline common/vehicle_constants_manager/package.xml:20]: cc @KeisukeShima We have to add a test case. I'll create an issue in https://github.com/tier4/pre-commit-hooks-ros.

**Comment by kenji-miyake** [comment]
> No concern from me but the packages that were ignored now need to be formatted properly.
Let me know if I can help you with that.

@maxime-clem Could you double check my changes? :pray: 
image

Some Markdown headers have changed, but I believe it's not a big problem.

File: simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp [hunk 2]
```diff
@@ -263,13 +260,9 @@ TEST_P(TestSimplePlanningSimulator, TestIdealSteerVel)
...
-  "IDEAL_STEER_ACC_GEARED",
-  "DELAY_STEER_VEL",
-  "DELAY_STEER_ACC",
-  "DELAY_STEER_ACC_GEARED",
+const char VEHICLE_MODEL_LIST[] = {
+  "IDEAL_STEER_VEL", "IDEAL_STEER_ACC", "IDEAL_STEER_ACC_GEARED",
+  "DELAY_STEER_VEL", "DELAY_STEER_ACC", "DELAY_STEER_ACC_GEARED",
 };
 
 INSTANTIATE_TEST_CASE_P(
```
Comment (maxime-clem) [inline simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp:268]: I do not know why but this code no longer builds.
The following changes allow the build to succeed.
```suggestion
const std::string VEHICLE_MODEL_LIST[] = {
  "IDEAL_STEER_VEL", "IDEAL_STEER_ACC", "IDEAL_STEER_ACC_GEARED",
  "DELAY_STEER_VEL", "DELAY_STEER_ACC", "DELAY_STEER_ACC_GEARED",
};

INSTANTIATE_TEST_SUITE_P(
```

Comment (kenji-miyake) [reply simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp:268]: Thank you.
Hmm, I've changed here following the `cpplint`'s advice.
I'll revert this change and add `// NOLINT`.

Comment (kenji-miyake) [reply simulator/simple_planning_simulator/test/test_simple_planning_simulator.cpp:268]: And I guess `INSTANTIATE_TEST_CASE_P` has no problem.

**Review by maxime-clem — APPROVED** [review]

**Pull request merged**